src/extension/module.js

```diff
@@ -1246,9 +1247,10 @@
               text: (titleI18n && titleI18n[language]) || title,
               action: () => {
                 if (url) {
-                  const target = url.startsWith('/')
+                  const baseTarget = url.startsWith('/')
                     ? new URL(url, `https://${innerHost}/`)
                     : url;
+                  const target = passReferrer ? `${baseTarget}?Referrer=${encodeURIComponent(location.href)}` : baseTarget;
```

Comment:
add the referrer via `URL.searchParams` otherwise you might have two `?` in the string.

Reply:
Should be fixed.